src/styles/_mixin.scss

```diff
@@ -0,0 +1,17 @@
+@mixin square($size) {
+  width: $size;
+  height: $size;
+  border-radius: 50%;
+}
+
+@mixin postionCenter($width-hand, $heigth-hand, $bgcolor-hand) {
```

Comment:
Why mixin position includes color? position have only X Y and some times Z, but never have color